### DISCUSSION TITLE
New to Github Education Community

### DISCUSSION THREAD

**Discussion by AbdurRaffay123** [community/community#139259] [New to GitHub]
Hey there, i hope you're doing well. I Would love to learn about this community and how can make my myself beneficial for others? And how can i make myself benefit from it?
Thanks

**Comment by A-S-W-I-N-S-P-A-R-K-Y** [comment]
Here's what you can expect moving forward ⏩

Your input will be carefully reviewed and cataloged by members of our product teams. 
Due to the high volume of submissions, we may not always be able to provide individual responses.
Rest assured, your feedback will help chart our course for product improvements.
Other users may engage with your post, sharing their own perspectives or experiences. 
GitHub staff may reach out for further clarification or insight. 
We may 'Answer' your discussion if there is a current solution, workaround, or roadmap/changelog post related to the feedback.












good

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @AbdurRaffay123 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
